crates/bevy_ui/src/geometry.rs

```diff
@@ -404,6 +404,13 @@ impl DivAssign<f32> for Size {
 mod tests {
     use super::*;
 
+    #[test]
```

Comment:
This test seem to be trivial. Do we need it?

Reply:
We can delete it, it's just covering the case and increases the code coverage. :)

Reply:
I'm going to bed (so I can only change it tomorrow), please let me know if I should remove the test. :3

It depends on your opinions. :3

Reply:
Not a fan of adding a test for the pure purpose of increasing coverage without testing non-trivial functionality. All it does is increase LoC count without adding anything.

I approved the PR as this isn't a blocker. Just voicing my opinion here.

Reply:
I've removed it, thanks for your feedback. :)